packages/guides/src/Handlers/ParseDirectoryHandler.php

```diff
@@ -98,14 +101,23 @@ private function getDirectoryIndexFile(
             $hashedContentFromFilesystem[$itemFromFilesystem['basename']] = true;
         }
 
-        foreach (self::INDEX_FILE_NAMES as $indexName) {
+        if ($indexName !== null) {
             $indexFilename = sprintf('%s.%s', $indexName, $extension);
             if (isset($hashedContentFromFilesystem[$indexFilename])) {
                 return $indexName;
             }
+        } else {
+            foreach (self::INDEX_FILE_NAMES as $indexName) {
+                $indexFilename = sprintf('%s.%s', $indexName, $extension);
+                if (isset($hashedContentFromFilesystem[$indexFilename])) {
+                    return $indexName;
+                }
+            }
+
+            $indexName = self::INDEX_FILE_NAMES[0];
```

Comment:
This seems to be invalid to me... you fetch the first value, but we tried more. Why not include all valid option in the exception?

Reply:
Good point, included them all now